src/main/java/frc/robot/Robot.java

```diff
@@ -159,12 +140,10 @@ public void autonomousInit() {
 
   @Override
   public void autonomousPeriodic() {
-    CommandScheduler.getInstance().run();
```

Comment:
How will our autonomous code execute if this is gone?

Reply:
The CommandScheduler is called to run in `roboticPeriodic()` no need to call it here or `teleopPeriodic()`

https://github.com/PearceRobotics/infinite-recharge-2020/pull/87/files#diff-4f7a983d6f9258e953cedb65c8544977L179

Reply:
>However, there is one exception: users must call CommandScheduler.getInstance().run() from the robotPeriodic() method of their Robot class. If this is not done, the scheduler will never run, and the command framework will not work. The provided command-based project template has this call already included.

from: https://docs.wpilib.org/en/latest/docs/software/commandbased/command-scheduler.html#using-the-command-scheduler